docs/logs/dedicated-pipeline/DedicatedLogging/DedicatedLoggingServiceCollectionExtensions.cs

```diff
@@ -37,7 +37,9 @@ public static IServiceCollection AddDedicatedLogging(
         return services;
     }
 
+#pragma warning disable CA1812 // Avoid uninstantiated internal classes
```

Comment:
nit: in a follow up PR; consider disabling these warnings for the entire docs directory or in the csproj file. The goal being to keep the examples simple.

Reply:
I think analysis should be disabled or significantly reduced for the docs and maybe the examples directory. They're purely for instructional purposes. All the `#pragma`s and `#if`s add noise that are not relevant to understanding the examples.